web/components/Layout.module.scss

```
@@ -1,13 +1,18 @@
 .container{
-    padding-bottom: 18vh;
-    align-content: center;
+    padding-bottom: 20vh;
     max-width: 1024px;
     margin: auto;
-    gap: 80px;
 }
 
+.flex{
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    width: 100%;
+}
 @media only screen and (max-width: 1024px) {
     .container{
        width: 100vw; 
     }
-}
\ No newline at end of file
+
+}
```

It's good practice to leave a blank line at the end of the file.
```suggestion
}

```